binary/type.proto

```diff
@@ -134,14 +140,9 @@ message Type {
     }
 
     message Map {
-
-        repeated KeyValue key_values = 1;
-        Variation variation = 2;
-
-        message KeyValue {
-            Type key = 1;
-            Type value = 2;
-        }
+        Type key = 1;
+        Type value = 2;
+        Variation variation = 3;
```

Comment:
Not sure what these changes for the Map are? It looks like the KV is no longer repeated?

Reply:
It was a mistake. This is for the type definition, not the literal value. The type definition just needs to list the two types.

Reply:
To clarify, the original definition was a mistake, not the change.